docs/Installation/BuildSystem.md

```diff
@@ -405,6 +306,148 @@ available to build with `make` or `ninja`:
   - Install targets that have been built to the `CMAKE_INSTALL_PREFIX`. Doesn't
     try to build anything else.
 
+# Editing the build system
+
+\note When editing `CMakeLists.txt` files, it is conventional to
+indent multiline commands by two spaces (except for the first line),
+and to separate most commands by blank lines.
+
+## Adding Source Files {#adding_source_files}
+
+SpECTRE organizes source files into subdirectories of `src` that are
+compiled into libraries.  To add a new source file `FILE.cpp` to an
+existing library in `src/PATH/DIR`, just edit
+`src/PATH/DIR/CMakeLists.txt` and add `FILE.cpp` to the list of files
+in
+```
+set(LIBRARY_SOURCES
```

Comment:
this is not true anymore.  we have `spectre_target_sources` and `spectre_target_headers`